inputs_topic : "mpc_inputs"
trajectory_topic : "mpc_trajectory"
metadata_topic : "mpc_metadata"